README.md

```diff
@@ -89,10 +89,14 @@ If you want to report a bug, first, thanks a lot. To help us, please indicate yo
 
 - Fork and clone
 - Master is usually unstable, checkout to a tag to have a stable state of the app
-- `npm install && npm run dev`
-- Enable dev mode in the app in the settings view to show DevTools
 
-Webpack will watch for JSX and SASS changes.
+- Install the latest version of electron by running `npm install -g electron`
+- NPM will install electron globally. You can use it on every of your electron projects now.

```

Comment:
> You can use it on every of your electron projects now.

I'd rephrased that. Latest electron might not work for just about any electron project.
Something like "You can use electron globally now with `electron [appdir]` command." or something like that,